sshd-core/src/main/java/org/apache/sshd/client/channel/ChannelSession.java

```diff
@@ -48,6 +53,7 @@ public class ChannelSession extends AbstractClientChannel {
 
     private CloseableExecutorService pumperService;
     private Future<?> pumper;
+    private Map<String, Object> env = new LinkedHashMap<>();
```

Comment:
Is there any special reason why you are using a `LinkedHashMap` rather than a `HashMap` ? Does the order of the keys matter in any way ?